homeassistant/components/aprilaire/strings.json

```diff
@@ -24,6 +24,14 @@
         "name": "Thermostat"
       }
     },
+    "humidifier": {
+      "humidifier": {
+        "name": "Humidifier"
+      },
+      "dehumidifier": {
+        "name": "Dehumidifier"
+      }
+    },
```

Comment:
These strings are provided by the `humidifier` entity component itself.

Reply:
Got it, thanks!

Reply:
You can remove them completely as you have set the device class and it will use the name from the entity component automatically. I'm sorry if I wasn't precise.

Reply:
@jpbede I tried that first and it did not pick up the name from the entity component automatically.

Reply:
Ok, my bad, this is not supported by the `humidifier` entity component.